### PR TITLE
Factor in DX for comparison with Go

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Good DX is a decisive factor for many people, so we should mention it.

### CODE REVIEW

File: site/docs/resources/comparison.md [hunk 1]
```diff
@@ -150,6 +150,11 @@ As a natively compiled language, Go outperforms TypeScript at raw CPU speed by s
 This is much less relevant when you write a bot because most of the time is spent waiting for the network, but eventually, it will start to matter how fast your bot can parse JSON.
 Go can be a better choice in these cases.
 
+Another metric is developer experience (DX).
+In general, Go and TypeScript are both known to have extremely good tooling and editor support.
+However, for the Bot API in particular, grammY is significantly better than any library written in Go.
```
Comment (rojvv) [inline site/docs/resources/comparison.md:155]: Why not specify a reason?

Comment (rojvv) [reply site/docs/resources/comparison.md:155]: Maybe something like "because of TypeScript"?

Comment (KnorpelSenf) [reply site/docs/resources/comparison.md:155]: True, I'll add that

**Review by trgwii — APPROVED** [review]
LGTM

**Review by rojvv — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]

File: site/docs/uk/resources/comparison.md [hunk 2]
```diff
@@ -150,6 +150,14 @@ TypeScript дозволяє ітераційно змінювати кодову
 Це набагато менш актуально, коли ви пишете бота, тому що більша частина часу витрачається на очікування мережі, але з часом почне мати значення, як швидко ваш бот може розбирати JSON.
 У таких випадках Go може бути кращим вибором.
 
+Інший показник - це досвід розробника (DX).
+Загалом відомо, що Go та TypeScript мають надзвичайно хороший інструментарій та підтримку редакторів.
+Однак, зокрема, для Bot API, grammY значно краща за будь-яку бібліотеку, написану на Go.
+Це головним чином завдяки розширеній системі типів TypeScript, яку grammY використовує у витончений спосіб.
+Як результат, ви можете досліджувати API ботів в інтерактивному режимі прямо з вашого редактора.
```
Comment (niusia-ua) [inline site/docs/uk/resources/comparison.md:157]:
```suggestion
Як результат, ви можете досліджувати Bot API в інтерактивному режимі прямо з вашого редактора.
```

**Review by niusia-ua — CHANGES_REQUESTED** [review]

**Review by niusia-ua — APPROVED** [review]

**Review by MasedMSD — APPROVED** [review]

**Review by quadratz — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@rojvv what is this waiting for?

**Comment by rojvv** [comment]
It is waiting for the Russian label to be renamed to `:ru: RU`.

**Comment by rojvv** [comment]
Done.

**Pull request merged**